CHANGES.md

```diff
@@ -6,6 +6,10 @@ This changelog is intended for _humans_ and follows many of the principles from
 Changes for this project _do not_ currently follow the [Semantic Versioning rules](https://semver.org/spec/v2.0.0.html).
 Instead, changes appear below grouped by the date they were added to the workflow.
 
+# 23 December 2025
+
+ - Add frequency and fitness estimates for amino acid haplotypes for each subtype and geographic resolution. See [#26](https://github.com/nextstrain/forecasts-flu/pull/26) for details.
+
```

Comment:
> Forecasts viz hardcodes the data provenance and does not provide an interface to select an alternate provenance.

Do you mean the "enabled by [gisaid logo]"? That's in `viz/index.html`, you can just take it out. The footer ("we gratefully acknowlege...") is also there.